integration/apps/hpl_netlib/README.md

```diff
@@ -30,10 +30,6 @@ This would normally be done by setting the following environment variables
 
     $ export KMP_HW_SUBSET=1t # forces 1 thread per core to be used
     $ export OMP_NUM_THREADS=44 # number of cores per node
-    $ export MKL_NUM_THREADS=44 # matches with above, effective if linked against MKL.
-
-MKL_NUM_THREADS is used by the MKL libraries to determine how many threads
-are used per rank.
```

Comment:
Was this intentionally removed?

Reply:
It was. Since this is the Netlib version and not the MKL version. A portion in the README.md was also referring to MKL, I removed that in the commit I will push now.